src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -110,6 +110,78 @@ def load_arguments(self, _):
+        c.argument('schema_name',
+                   help='The name of the schema resource.')
+        c.argument(
+            'api_id', arg_group='Schema',
+            help='API identifier. Must be unique in the current API Management service instance. Non-current revision has ;rev=n as a suffix where n is the revision number.',
+            required=True)
+        c.argument('schema_path', help='File path specified to import schema of the API.')
+        c.argument('schema_content', help='Json escaped string defining the document representing the Schema')
+        c.argument(
+            'schema_id', arg_group='Schema',
+            help='Schema identifier. Must be unique in the current API Management service instance. Non-current revision has ;rev=n as a suffix where n is the revision number.',
+            required=True)
+        c.argument(
+            'schema_type', arg_group='Schema',
+            help='Schema content type. Must be a valid a media type used in a Content-Type header as defined in the RFC 2616. Media type of the schema document (e.g. application/json, application/xml).',
+            required=True)
+        c.argument(
+            'resource_type', arg_group='Schema',
+            help='The type of the resource. E.g. "Microsoft.Compute/virtualMachines" or "Microsoft.Storage/storageAccounts".')
+        c.argument('if_match', help='ETag of the Entity.')
+
+    with self.argument_context('apim api schema delete') as c:
+        c.argument('service_name', options_list=['--service-name', '-n'],
+                   help='The name of the API Management service instance.')
+        c.argument(
+            'api_id', arg_group='Schema',
+            help='API identifier. Must be unique in the current API Management service instance. Non-current revision has ;rev=n as a suffix where n is the revision number.',
+            required=True)
+        c.argument(
+            'schema_id', arg_group='Schema',
+            help='Schema identifier. Must be unique in the current API Management service instance. Non-current revision has ;rev=n as a suffix where n is the revision number.',
+            required=True)
+        c.argument('if_match', help='ETag of the Entity.')
+
+    with self.argument_context('apim api schema show') as c:
+        c.argument('service_name', options_list=['--service-name', '-n'],
+                   help='The name of the API Management service instance.')
+        c.argument(
+            'api_id', arg_group='Schema',
+            help='API identifier. Must be unique in the current API Management service instance. Non-current revision has ;rev=n as a suffix where n is the revision number.',
+            required=True)
+        c.argument(
+            'schema_id', arg_group='Schema',
+            help='Schema identifier. Must be unique in the current API Management service instance. Non-current revision has ;rev=n as a suffix where n is the revision number.',
+            required=True)
```

Comment:
I see that these parameters have been repeated many times. Could we consider abstracting them into `CLIArgumentType` for reuse?

Reply:
I will check on this

Reply:
Hi, Azure pipeline is failing I think issue is not with my current code changes.